app/src/main/res/values/color_palette.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="primary_color">@color/oppia_primary</color>
```

Comment:
Shouldn't this point to a specific color rather than oppia_primary? Ditto for the others.

Reply:
Yes, but the reason I didn't change it yet is because most activities(other than the ones I changed in this PR) are still referencing to older pre-defined colors in `colors_def.xml` like (oppia_primary in color_defs.xml). I thought it will be easier to track the changes if we change those default values after we have implemented dark mode through `component_colors.xml` to most of the layouts. As then we will only have to change the values in `color_palette.xml` rather than many files.
@BenHenning What are your views on this? If you suggest otherwise I will rename existing colors in `color_defs.xml` in this PR only.

Reply:
Might it be possible to introduce a temporary fourth file, maybe colors_migrating.xml that have *all* colors that need to be categorized and/or renamed? That would allow you to ensure each of the other color files is setup exactly the way we want it to be long-term without having to make any concessions.

Reply:
Sounds good. Added `colors_migrating.xml`.

Reply:
Thanks. Can this file at least be updated to point to the correct color? It doesn't really make sense for the new files to reference colors_migrating.xml since only migrated colors should go in these files.